services/director/src/simcore_service_director/registry_proxy.py

```diff
@@ -156,7 +156,9 @@ async def _auth_registry_request(  # noqa: C901
         return (resp_data, resp_headers)
     if auth_type == "Basic":
         # basic authentication should not be since we tried already...
-        resp_wbasic = await getattr(session, method.lower())(url, auth=auth, **kwargs)
+        resp_wbasic = await getattr(session, method.lower())(
+            str(url), auth=auth, **kwargs
```

Comment:
MINOR: we normally stringify using f-strings*: `f"{url}"` since it performs a bit faster

* which has nothing to do with f-words ;-)

Reply:
https://medium.com/@DahlitzF/pythons-f-strings-vs-str-e22995cefef6